.buildkite/pipeline.yml

```diff
@@ -1,5 +1,46 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
-steps:
-  - label: "Example test"
-    command: echo "Hello!"
+env:
+  SETUP_GVM_VERSION: 'v0.5.0'
+  GO_VERSION: "1.19.10"
+  LINUX_AGENT_IMAGE: "family/core-ubuntu-2004"
+  WINDOWS_AGENT_IMAGE: "family/ci-windows-2022"
+
+  steps:
+  - label: ":golangci-lint: Lint"
+    key: lint-test
+    command: ".buildkite/scripts/lint_test.sh"
+    agents:
+      provider: "gcp"
+      image: "${LINUX_AGENT_IMAGE}"
```

Comment:
Do these steps require docker to run their tests/lint? 
Could it be used just the golang image for these steps ?